### DISCUSSION TITLE
rome lsp-proxy should support --config-path

### DISCUSSION THREAD

**Discussion by dsully** [rome/tools#4393] [Suggestions]
In #4158 the `--config-path` option is added to `rome check`. This should also be added to the `lsp-proxy` subcommand, such that a global `rome.json` can be used via the LSP.

I can attempt to make this change if it's a PR that would be accepted.

Thanks

**Comment by ematipico** [comment]
Could you please help us understand why you would need that option? The LSP has a different resolution from the CLI. If you could share some use case, that would be helpful


BTW: `--config-path` is a[ global option ](https://docs.rome.tools/cli/#--config-path) that you can pass to any command.

**Comment by dsully** [comment]
I want to use `rome` as a language server in Neovim for JSON that isn't part of a project that would have a `rome.json` file in it.

FWIW, `rome help` or `rome lsp-proxy --help` doesn't show `--config-path` as an option at all.

It looks like `rome lsp-proxy --config-path /path/to/rome.json` starts up - I need to hook it up to Neovim later today to see if it is doing the right thing.
